app/code/Magento/CatalogInventory/Plugin/MassUpdateProductAttribute.php

```diff
@@ -81,11 +81,10 @@ public function __construct(
      * Around execute plugin
      *
      * @param Save $subject
-     * @param callable $proceed
      *
-     * @return \Magento\Framework\Controller\ResultInterface
+     * @return array
      */
-    public function aroundExecute(Save $subject, callable $proceed)
+    public function beforeExecute(Save $subject)
```

Comment:
We don't really need to change arguments for "save", so it's better to use `void` return type here and not use return statement at all 
```suggestion
    public function beforeExecute(Save $subject): void
```